src/components/table-hoc/TableWithFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {keys} from 'ts-transformer-keys/index';
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
+import {ReduxConnect} from '../../utils/ReduxUtils';
+import {FilterBoxConnected} from '../filterBox/FilterBoxConnected';
+import {IFilterState} from '../filterBox/FilterBoxReducers';
+import {ITableHOCOwnProps} from './TableHOC';
+
+export interface ITableWithFilterConfig {
+    matchFilter?: (filterValue: string, datum: any) => boolean;

```

Comment:
Could `matchFilter`  be generic to allow library users to inject their own type? `matchFilter<T = any>(filterValue: string, datum: T)`

Reply:
Cool stuff, did not notice this, I'm fine with it :)